bin/fun-init.js

```diff
@@ -46,7 +46,8 @@ program
   .usage('[options] [location]')
   .description('Initializes a new fun project.')
   .option('-o, --output-dir [path]', 'where to output the initialized app into', '.')
-  .option('-n, --name [name]', 'name of your project to be generated as a folder', 'fun-app')
+  .option('-n, --name [name]', 'name of your project to be generated as a folder', '')
+  .option('-m, --merge [merge]', 'whether the template.[yml|yaml] file in the template project is merged with the existing template.[yml|yaml] of the local file system', false)
```

Comment:
merge into the template.[yml|yaml] file if it already exist ？

Reply:
OK